Dockerfile.apidocs

```diff
@@ -0,0 +1,51 @@
+# Sentry api documentation generation environment
+#
+# Instructions:
+#
+#   Build the container:
+#     $ docker build -t sentry:apidocs -f Dockerfile.apidocs .
+#   Run the container:
+#     $ docker run --rm -v $(pwd):/usr/src/output sentry:apidocs
+#
+# The container will dump the generated documentation in markdown and JSON
+# formats into the /usr/src/output directory which you should mount as a volume
+#
+FROM python:2.7.11-slim
+
+RUN groupadd -r redis --gid=998 && useradd -r -g redis --uid=998 redis \
+    && groupadd -r postgres --gid=999 && useradd -r -g postgres --uid=999 postgres
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        clang \
+        curl \
+        g++ \
+        gcc \
+        git \
+        libffi-dev \
+        libjpeg-dev \
+        libpq-dev \
+        libxml2-dev \
+        libxslt-dev \
+        libyaml-dev \
+        llvm-3.5 \
+        bzip2 \
+        make \
+        unzip \
+    && rm -rf /var/lib/apt/lists/*
+
+# Sane defaults for pip
+ENV PIP_NO_CACHE_DIR off
+ENV PIP_DISABLE_PIP_VERSION_CHECK on
+ENV PYTHONUNBUFFERED 1
+
+# Install Redis
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y --no-install-recommends redis-server
```

Comment:
Would just add this into the list above. No reason to have it separate.

Reply:
Good point.